Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -1,13 +1,22 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     // TODO: make a struct and add validation and other needed methods (if needed)
     public class Credentials
     {
         public string Username;
         public string Password;
 
+        public string ToFile { get { return $"{Username},{Password}"; } }
```

Comment:
Here you don't do anything with a file, so it looks deceiving.
It's just converting Credentials ToString.
``ToString()`` is a virtual method that you can override, so please do :). Also, don't forget that you can use expressions to simplify it
```
public string ToFile => ${Username},{Password}";
```